### PR TITLE
Added PLCrashReporter to log crashes

### PR DESCRIPTION
**Pull request by mneuwert**
## Description
- Integrated PLCrashReporter as swift package using SPM
- At startup it is checked, if there is pending crash report from previous app session. Crash report is printed as text into the current log.

## Related Issue
#702 

## Motivation and Context
We want to enable users to report bugs related to crashes by sending us logs which will include crash reports.

## How Has This Been Tested?
- Changed the app so that it crashes when camera upload action is triggered by calling `fatalError()`
- Checked the log upon next app launch if it has crash report included.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ownCloud/AppDelegate.swift [hunk 1]
```diff
@@ -206,3 +208,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		}
 	}
 }
+
+import CrashReporter
```
Comment (hosy) [inline ownCloud/AppDelegate.swift:212]: Wouldn't it be cleaner to add the import statement on the top of the file?

Comment (mneuwert) [reply ownCloud/AppDelegate.swift]: @hosy moved but not sure if it really contributes to cleanliness

**Comment by jesmrec** [comment]
These are the crash traces i found:

Screenshot 2020-07-08 at 09 16 18

is this correct? i guess so, just to confirm

**Comment by jesmrec** [comment]
then, crash reports are there, from my side it is OK to move forward. I did not test whether the info included in the reports is correct or not, only that causing a crash makes the app write crash errors.

If i'm missing something, let me know. If not, this is approved

**Pull request merged**